pkg/event_handler/github_event_notifier.go

```diff
@@ -27,6 +27,15 @@ var workflowTranslationToBitbucketMap = map[string]string{
 	"Error":     "STOPPED",
```

Comment:
wdyt about rename the file ?

Reply:
Mayeb let's open an issue to move all those maps to be implemnted by the client, then will change the file name

Reply:
@omri2001 agree, can you do it ?